File: Java/백준/Bronze/25600. Triathlon/Triathlon.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int q = in.nextInt();
        int ans = 0;
        int max = 0;
        for (int i =0; i<q; i++){
            int a = in.nextInt();
            int b = in.nextInt();
            int c = in.nextInt();
            int score = a*(b+c);
            if(a==b+c){
                score*=2;
            }
            max = max>score?max:score;
        }
        System.out.println(max);
    }
}

File: C++11/백준/Bronze/25600. Triathlon/Triathlon.cc
#include <iostream>
using namespace std;
int main(){
    int q,a,b,c;
    int max =  0;
    cin >> q;
    for(int i =0; i<q; i++){
        cin >> a>> b>> c;
        int score = a*(b+c);
        if(a==b+c){
            score *= 2;
        }
        max = max>score?max:score;
    }
    cout << max;
}